docs/source/user-guide/general/metrics_guide.rst

```diff
@@ -0,0 +1,182 @@
+Deepchecks.
+The assessment of how well the model performs should drive informed decision making regarding the usage of that model,
+so we have to make sure that we measure the right things, i.e. the criteria that represnt tha key factors in the
+decision making process.
+
+**Structure:**
+
+* `How can I use metrics with Deepchecks? <#how-can-i-use-metrics-with-deepchecks>`__
+* `Default metrics <#default-metrics>`__
+* `Alternative metrics <#alternative-metrics>`__
+* `List of Supported Strings <#list-of-supported-strings>`__
+* `Custom Metrics <#custom-metrics>`__
+
+
+How can I use metrics with Deepchecks?
+======================================
+The metrics used in deepchecks are in accordance with sklearn scorer API for tabular checks and with Torch Ignite metric
+API for vision checks.
+To use metrics with Deepchecks suites and checks you can use the defaults, that fits the most common usecase, or pass
+alternative scorers that are more accurate for your specific usecase.
+
+Default metrics
+===============
+All of our relevant checks come with default metrics.
+
+The default scorers by task type are:
+
+Tabular:
+
+Classification, binary or multiclass averaged over the classes:
+
+*   Accuracy
+*   Precision
+*   Recall
+
+Multiclass classification per class:
+
+*   F1
+*   Precision
+*   Recall
+
+Regression:
+
+*   Negative RMSE (negative so larger is better)
+*   Negative MAE
+*   R2
+
+Vision:
+
+Classification:
+
+*   Precision
+*   Recall
+
+Object detection:
+
+*   Mean average precision
+*   Mean average recall
+
+To run a check with the default metrics, run it without passing the scorers.
+
+:doc:`ClassPerformance </checks_gallery/vision/model_evaluation/plot_class_performance>`:
+
+.. code-block:: python
+
+    check = ClassPerformance()
+    result = check.run(train_ds, test_ds, mnist_model)
+
+
+Alternative metrics
+===================
+Sometimes the defaults aren’t good enough to describe the specifics of the problem.
+If this is the case, you can pass a list of metrics or a dict in the format {metric name: metric} as a parameter to the
+check.
+
+The metrics on the list can be existing
+`Ignite metrics <https://pytorch.org/ignite/metrics.html#complete-list-of-metrics>`__ or
+`Sklearn scorers <https://scikit-learn.org/stable/modules/model_evaluation.html>`__ or
+or a string from Deepchecks' `supported strings <#list-of-supported-strings>`__ or your own implementations.
+
+.. code-block:: python
+
+   train_ds, test_ds = adult.load_data(data_format='Dataset', as_train_test=True)
+   model = adult.load_fitted_model()
+
+   scorer = ['precision_per_class', 'recall_per_class', 'fnr_macro']
+   check = TrainTestPerformance(scorers=scorer)
+   result = check.run(train_ds, test_ds, model)
+   result
+
+
+List of Supported Strings
+=========================
+.. list-table:: Supported Strings
+   :widths: 25 75
+   :header-rows: 1
+
+   * - String
```

Comment:
I would name all metrics, we have only like 10 of them across all tasks. If divided into categories and mention averaging method in a different section I think it would be fine.
Anyway not that critical to me

Reply:
@noamzbr I copied the list from the scorers module

Reply:
@noamzbr @Nadav-BarakIMO we should have the full list so the user can copy-paste the strings without too much thought.

Reply:
Here you wrote the combinations only for fpr fnr tnr, not for precision recall and f1. If you had done that, you'd have gotten an extremely long list. I still think that listing all combinations can't be done, but it must be made clear what metrics have these combinations (fpr fnr tnr precision recall f1) and which don't (e.g. accuracy) and which have a different mechanism (e.g. auc), when the latter need to be added here.

Also, still think we should do my points 1&2 from the start of this thread